nix/pkgs.nix

```diff
@@ -0,0 +1,28 @@
+let
+  sources = import ./sources.nix {};
+  haskellNix = import sources.haskellNix {};
+
+  nixpkgsSrc = haskellNix.sources.nixpkgs-unstable;
+  nixpkgsArgs = haskellNix.nixpkgsArgs;
+
+  native = import nixpkgsSrc nixpkgsArgs;
+
+  x86_64-static = import nixpkgsSrc (nixpkgsArgs // {
+    crossSystem = native.lib.systems.examples.musl64;
+  });
+
+  aarch64-static = import nixpkgsSrc (nixpkgsArgs // {
+    crossSystem = native.lib.systems.examples.aarch64-multiplatform-musl;
+  });
+
+  aarch64-darwin = import nixpkgsSrc (nixpkgsArgs // {
+    crossSystem = native.lib.systems.examples.aarch64-darwin;
+  });
```

Comment:
I don't quite get why there is plumbing for cross-compiling for Darwin AArch64 here, but it's not used during release builds as far as I can tell.

Reply:
I'm working on that on my local machine, but wanted to release this now that I have the linux binaries working already

Reply:
Darwin aarch64 Is about the new Apple M1 so it is definitely needed

Reply:
@Fixmetal you can already get it from homebrew